README.md

```diff
@@ -80,19 +90,25 @@ class AnyPageSpec
 }
 ```
 
-To run the tests do,
+To run the tests you can either do,
+
+```shell script
+sbt a11y:test  
+```
+
+Or alternatively you can run,
 
 ```shell script
-sbt a11yTest
+sbt accessibilityAssessment
 ```
 
 The above command is a swap-in replacement for `sbt test`.
 
 Once you have added these tests, you will need to modify your `SbtMicroserviceJobBuilder` job definition in
 build jobs to ensure the tests work in CI.
 
-To do this, simply change `test` to `a11yTest` in the line similar to `.withTests("test it:test")`. If you do not 
-already invoke `withTests` directly, add `.withTests("a11yTest it:test")` to your `SbtMicroserviceJobBuilder` job
+To do this, simply change `test` to `a11y:test` in the line similar to `.withTests("test it:test")`. If you do not 
+already invoke `withTests` directly, add `.withTests("a11y:test it:test")` to your `SbtMicroserviceJobBuilder` job
 definition to replace the default behaviour.
```

Comment:
@kyle-bowden I think this won't be true anymore, now I was expecting it would be `withTests("test a11y:test it:test")`

Reply:
I have made the change, thank you.